release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1122,9 +1123,9 @@ For more information, see xref:../updating/updating-restricted-network-cluster/r
 
 [discrete]
 [id="ocp-4-13-ocm-nodeip-configuration-vsphere"]
-=== The `nodeip-configuration` service is now enabled on a vSphere user-provisioned infrastructure cluster
+=== The nodeip-configuration service is now enabled on a vSphere user-provisioned infrastructure cluster
```

Comment:
Definitely not a blocker, but after GA, it would be good to follow up with the original writers to reformat these types of headings. Something like "Node IP configuration sevice" instead of the object names. Not a big deal, and definitely not important now.